### DISCUSSION TITLE
Allocating More Memory to Server

### DISCUSSION THREAD

**Discussion by ethanic17** [itzg/docker-minecraft-server#1237] [Q&A] [answered]
Hello, I just installed the Paper version of the docker-mc-server on my Raspberry Pi and I'm having trouble increasing the memory to 2GB. I'm a noob when it comes to all of this, so I apologize in advance if this is a relatively simple fix or if I just overlooked something. I already ran 
```
docker run -d -v /path/on/host:/data \
    -e TYPE=PAPER \
    -p 25565:25565 -e EULA=TRUE --name mc itzg/minecraft-server
```
but didn't include the memory code (parameter?) ` -e MEMORY=2G` and I'm wondering if there's a way to change the memory allocation after I created the server or if I just have to re-do everything all over again. Once again, I'm kind of new to all of this so excuse me if this sounds really simple or if I just overlooked something.

**Comment by itzg** [comment, marked as answer]
Yes you'll need to recreate the container, which is trivial since you're using a data volume mapping.

**Reply by ethanic17** [reply]
ah sweet, thank you, and one more thing, since the mc server is running as a docker, does that mean the server is running at root/administrator level? I'm planning to port forward and open up the server and was wondering if there were any security risks involved with it running at the root level and if someone with malicious intent was able to gain backdoor access to not just the mc server, but also my local files and potentially other devices on my network.

**Reply by itzg** [reply]
After very initial setup, the container startup demotes to a non-root user.

**Reply by ethanic17** [reply]
Gotcha, thanks.
